tests/integration/scenarios/filter_kubernetes/tests/test_filter_kubernetes_001.py

```
@@ -0,0 +1,143 @@
+
+
+class _KubeApiHandler(http.server.BaseHTTPRequestHandler):
+    def do_GET(self):
+        payload = b"{}"
+        self.send_response(200)
+        self.send_header("Content-Type", "application/json")
+        self.send_header("Content-Length", str(len(payload)))
+        self.end_headers()
+        self.wfile.write(payload)
+
+    def log_message(self, format, *args):
```

Ruff (0.15.10)

[error] Function argument `format` is shadowing a Python builtin

(A002)